Bento/Views/CollectionViewCell.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+final class CollectionViewCell: UICollectionViewCell {
```

Comment:
Call we rename it to `CollectionViewContainerCell`? Just to make it more clear that's not an usual collection view cell. And consequently extend that to `TableViewCell`.